app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [REQUIRED] Kindly, enable `:confirmable` in order to allow email confirmation. This is a hard requirement. Please implement it before requesting another review.